src/_data/sites/yetty.json

```diff
@@ -0,0 +1,8 @@
+{
+	"url": "https://yetty.netlify.app/",
+	"name": "Yetty",
+	"description": "Yet another Eleventy starter kit for my (& your) new projects. Built with accessibility and performance in mind.",
+	"source_url": "https://github.com/ygoex/yetty",
+	"twitter": "ygoex",
+	"authoredBy": "Yolanda Gorriz Exposito"
```

Comment:
I think you can remove this `authoredBy` line. Per the "new issue template", it looks like `authoredBy` is an array of Twitter usernames if there is more than one site author.

> * `url`: The site’s production URL
> * `name`: Name of the site
> * `description`: A short text description of the site
> * `twitter`: Twitter username for the site or the site’s author.
> * `authoredBy`: An array of Twitter usernames of the site’s authors. Supplements the `twitter` entry. (Optional)
> * `source_url`: URL to the source code (Optional)

Reply:
👍 Done!